test/services/concerns/xccdf/rules_test.rb

```diff
@@ -47,6 +50,19 @@ def initialize(report_contents)
     assert_includes @mock.rules, rule
   end
 
+  test 'correctly saves value_checks' do
+    @mock.save_rules
+    rule1 = Rule.find_by(ref_id: 'xccdf_org.ssgproject.content_rule_network_ipv6_disable_interfaces')
+    rule2 = Rule.find_by(ref_id: 'xccdf_org.ssgproject.content_rule_sshd_set_idle_timeout')
+    value1 = ValueDefinition.find_by(ref_id: 'xccdf_org.ssgproject.content_value_sshd_required')
+    value2 = ValueDefinition.find_by(ref_id: 'xccdf_org.ssgproject.content_value_sshd_idle_timeout_value')
+
+    assert_includes rule2.value_checks, value1.id
+    assert_includes rule2.value_checks, value2.id
+    assert_equal 2, rule2.value_checks.length
+    assert_equal [], rule1.value_checks
```

Comment:
You don't set up a single context for this test...how do you actually verify things here?

Reply:
`@mock.save_value_definitions` is being done in the setup which is making sure `@value_definitions` is defined because it's used in `@mock.save_rules`. 
The purpose of this test is just to make sure that the `value_checks` column is being set up correctly during `@mock.save_rules`. To verify, I used specific rule and value ref_ids from the test file that's being used for these tests: `xccdf_report.xml`

Reply:
`rule1` is a rule in `xccdf_report.xml` that has no value checks and `rule2` is a rule in `xccdf_report.xml` that has 2 value checks.

Reply:
Okay, so this is just a simple assertion that the stuff is imported as it is in the fixture. You also need some tests for overriding existing stuff (see other files for examples).